common/f3d_materials.md

```diff
@@ -0,0 +1,193 @@
+# F3D Material Overview
+
+F3D materials offer an accurate way of recreating N64 graphics within blender. The materials contain all the inputs and outputs that N64 graphics using Nintendo's standard "Fast 3D" material system uses, with almost no concessions with regards to visual accuracy.
+
+F3D materials are can be modified in either a simple or full UI. Using the simplified UI will allow you to select from preset options, and change the applicable input variables as needed, while the full UI will unlock all F3D options available. Each section within the full UI represent an important chunk of the F3D material system, and should be understood to make full use of the N64s graphical capabilities.
+
+:::::{tab-set}
+
+::::{tab-item} Color Combiner & Sources
+
+Color and alpha values for pixels are determined using the color combiner. The color combiner takes various source value such as textures, solid colors, lights and for each pixel applies a formula of {math}`Color = (A-B)*C + D` to get a resultant value out for both color and alpha. This value is sent to the blender for combining the pixel and alpha with existing pixel colors or additional inputs such as fog.
+
+## Color Combiner
+In the F3D material inspector, the Combiner tab offers you a selection of 8 inputs in one cycle mode, and 16 in two cycle mode; by default one cycle mode is selected. The inputs in the left represent the RGB color that will be used for blending color, while the ones on the left aptly labeled "Alpha" will be for alpha blending.
```

Comment:
> by default one cycle mode is selected.

FTR this is not true when using fast64 with OoT

> blending

use "combining" instead since this is the combiner